2001-3000/LIT2892RepCh173.xml

```diff
@@ -58,14 +59,29 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <bibl>
                   <ptr target="bm:Chaine1913Rep"/>
                   <citedRange unit="item">173</citedRange>
+                  <note>
+                     <persName ref="PRS3082Chaine"/> was mistakenly identified this a malkǝʾ-hymn for
+                     <persName ref="PRS9848Walatta"/> in 
+                     <bibl>
+                        <ptr target="#bm:Chaine1913RepPart1"/>
```

Comment:
no `#` in ptr
`<ptr target="bm:Chaine1913RepPart1"/>`
also below

Reply:
Ok! Thank you!